docker/elasticsearch/README.md

```diff
@@ -0,0 +1,5 @@
+#### 2.0 Start Elasticsearch with docker
+If you familier with docker you can use the elasticsearch docker image with installed elasticsearch-vector-scoring plugin.
```

Comment:
consider linking out to https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html for more info